docs/source/indepth_tutorial/directed-search.ipynb

```diff
@@ -98,9 +98,9 @@
      "name": "stderr",
```

Comment:
<p>Above this cell, very briefly explain what is done/run in this cell (or in comments in the cell)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quaquel/EMAworkbench/pull/194/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>